ding_nodelist.module

```diff
@@ -507,16 +507,21 @@ function ding_nodelist_add_bottom_link_submit(&$form, &$form_state) {
  * Autocomplete function that gives the available nodes.
  */
 function ding_nodelist_autocomplete($types, $title = '') {
+  global $language;
   $types = explode('|', urldecode($types));
 
   // Fetch all published nodes.
   $nodes = db_select('node', 'n')
     ->fields('n', array('nid', 'title'))
     ->condition('n.type', $types, 'IN')
     ->condition('n.title', "%$title%", 'LIKE')
-    ->condition('n.status', NODE_PUBLISHED)
-    ->execute()
-    ->fetchAllKeyed();
+    ->condition('n.status', NODE_PUBLISHED);
+
+  if (module_exists('ding_language')) {
```

Comment:
Indent is wrong. Why not `->addTag('language')`?

Reply:
> Indent is wrong.

Can't agree. Here was broken original query so the next if() is indented correctly.

> ->addTag()

 adds unique identifier to a query which will be altered further, as we still can perform different code changes in ding_nodelist we can leave it as it is (IMHO).

Reply:
> adds unique identifier to a query which will be altered further, as we still can perform different code changes in ding_nodelist we can leave it as it is (IMHO).

Understood - then maybe?
```
if (module_exists('ding_language')) {
  ->addTag('language')
} 
```